application-tls-termination/README.md

```diff
@@ -0,0 +1,260 @@
+    --profile leaf \
+    --not-after 2160h \
+    --ca root-ca.crt \
+    --ca-key root-ca.key \
+    --no-password \
+    --insecure \
+    --san platform-pypi \
+    --san platform-pypi.default.svc.cluster.local \
+    --set "subject.organization=Howso Platform"
+```
+
+Create the corresponding Kubernetes secret:
+
+```bash
+kubectl -n howso create secret tls platform-pypi-server-tls --key platform-pypi-server-tls.key --cert platform-pypi-server-tls.crt
+```
+
+#### Platform UMS Server
+
+Create the certificate for the UMS server:
+
+```bash
+step certificate create platform-ums.default.svc.cluster.local platform-ums-server-tls.crt platform-ums-server-tls.key \
+    --profile leaf \
+    --not-after 2160h \
+    --ca root-ca.crt \
+    --ca-key root-ca.key \
+    --no-password \
+    --insecure \
+    --san platform-ums \
+    --san platform-ums.default.svc.cluster.local \
+    --set "subject.organization=Howso Platform"
+```
+
+Create the corresponding Kubernetes secret:
+
+```bash
+kubectl -n howso create secret tls platform-ums-server-tls --key platform-ums-server-tls.key --cert platform-ums-server-tls.crt
+```
+
+#### Platform UI (v2) Server
+
+Create the certificate for the UI:
+
+```bash
+step certificate create platform-ui-v2.default.svc.cluster.local platform-ui-v2-server-tls.crt platform-ui-v2-server-tls.key \
+    --profile leaf \
+    --not-after 2160h \
+    --ca root-ca.crt \
+    --ca-key root-ca.key \
+    --no-password \
+    --insecure \
+    --san platform-ui-v2 \
+    --san platform-ui-v2.default.svc.cluster.local \
+    --set "subject.organization=Howso Platform"
+```
+
+Create the corresponding Kubernetes secret:
+
+```bash
+kubectl -n howso create secret tls platform-ui-v2-server-tls --key platform-ui-v2-server-tls.key --cert platform-ui-v2-server-tls.crt
+```
+
+#### Platform API (v3) Server
+
+Create the certificate for the API v3 server:
+
+```bash
+step certificate create platform-api-v3.default.svc.cluster.local platform-api-v3-server-tls.crt platform-api-v3-server-tls.key \
+    --profile leaf \
+    --not-after 2160h \
+    --ca root-ca.crt \
+    --ca-key root-ca.key \
+    --no-password \
+    --insecure \
+    --san platform-api-v3 \
+    --san platform-api-v3.default.svc.cluster.local \
+    --set "subject.organization=Howso Platform"
+```
+
+Create the corresponding Kubernetes secret:
+
+```bash
+kubectl -n howso create secret tls platform-api-v3-server-tls --key platform-api-v3-server-tls.key --cert platform-api-v3-server-tls.crt
+```
+
+
+### Create Ingress Resources
+
+Take a look at the [manifests](./manifests/) for the services.  In this section we'll apply the ingress configuration service by service, and check that it is working.  If you hit issues, [turn on debug logs](#troubleshooting) in Traefik to see what is going wrong. 
+
+
+#### Platform PyPI Ingress
+
+Create the ingress resource for the platform-pypi service:
+
+```yaml
```

Comment:
why yaml here? (instead of bash; there are more in the file just commenting on one.)